src/main/resources/application.yaml

```diff
@@ -63,6 +63,8 @@ ftp:
       folder: DIVORCE
     - service: probate_backend
       folder: PROBATE
+    - service: sscs
```

Comment:
please check the s2s service name

Reply:
@jackmaloney do you know what an s2s service name is?

Reply:
logs suggested this was the service accessing send letter

Reply:
I'm reliably informed it's set here https://github.com/hmcts/sscs-evidence-share/blob/master/src/main/resources/application.yaml#L54 and is usually sscs